src/ui/ui-storyEditor.js

```diff
@@ -14,14 +14,33 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
     // a UI for editing block-based stories
     fluid.defaults("sjrk.storyTelling.ui.storyEditor", {
         gradeNames: ["sjrk.storyTelling.ui"],
+        model: {
+            // the metadata step is hidden when the edit story step is shown,
+            // and vice versa. This is represented using this value alone, for now.
+            editStoryStepVisible: true
+        },
+        modelListeners: {
+            "editStoryStepVisible": [{
+                this: "{that}.dom.storyEditStoryStep",
+                method: "toggle",
+                args: ["{change}.value"],
+                namespace: "manageEditStoryStepVisibility"
+            },
+            {
+                this: "{that}.dom.storyMetadataStep",
+                method: "toggle",
+                args: ["{change}.oldValue"],
```

Comment:
I understand you use `{change}.oldValue` is to retrieve the opposite of the current boolean value in `{change}.value`. However, it introduces a logic confusion that this action is controlled by the old model value rather than the current. Personally, I prefer to sacrifice a little coding convenience for clarifying the logic.

Reply:
I agree that this method is confusing. What I'd like is to have a way to say "the opposite of `{change}.value`", but without the `negate` transform I was beginning to create (my work on that can be found in [my WIP branch](https://github.com/BlueSlug/sjrk-story-telling/blob/fd1f8096dcbf1ddd98a484c7dacc130412d898bb/src/ui/transforms.js#L111-L126)), I'm not sure how best to approach this.

What do you think of adding a basic "not" function locally in this project, with the aim of using it in an expander and passing in `{change}.value`? This function can later be supplanted by the `negate` transform, whenever that ends up being incorporated into Infusion.

Reply:
I've just noticed @amb26's comment below suggesting to do something very similar